overlord/servicestate/quota_handlers.go

```diff
@@ -220,17 +220,8 @@ func ensureSnapServicesForGroup(st *state.State, grp *quota.Group, opts *ensureS
 			return err
 		}
 
-		// drop disabled services from the startup ordering
-		startupOrderedMinusDisabled := make([]*snap.AppInfo, 0, len(startupOrdered)-len(disabledSvcs))
-
-		for _, svc := range startupOrdered {
-			if !isDisabledSvc[svc.ServiceName()] {
```

Comment:
this can be done maybe by @anonymouse64 in one of its quota PRs but afaict this means we don't need code to precompute isDisabledSvc at all but tests need changes